core/include/detray/propagator/stepping_config.hpp

```diff
@@ -45,4 +46,30 @@ struct config {
     bool do_covariance_transport{true};
 };
 
+/// Print the stepping configuration
+DETRAY_HOST
+inline std::ostream& operator<<(std::ostream& out,
+                                const detray::stepping::config& cfg) {
+    out << "  Min. Stepsize         : "
+        << cfg.min_stepsize / detray::unit<float>::mm << " [mm]\n"
+        << "  Runge-Kutta tolerance : "
+        << cfg.rk_error_tol / detray::unit<float>::mm << " [mm]\n"
+        << "  Max. step updates     : " << cfg.max_rk_updates << "\n"
+        << "  Stepsize  constraint  : "
+        << cfg.step_constraint / detray::unit<float>::mm << " [mm]\n"
+        << "  Path limit            : "
+        << cfg.path_limit / detray::unit<float>::m << " [m]\n"
+        << std::boolalpha << "  Use Bethe energy loss : " << cfg.use_mean_loss
+        << "\n"
+        << "  Do cov. transport     : " << cfg.do_covariance_transport << "\n";
+
+    if (cfg.do_covariance_transport) {
+        out << std::boolalpha
```

Comment:
These kinds of operations modify the `ostream` state right? That might be unexpected by the user

Reply:
Hm, good point, will add ```std::noboolalpha```